src/tests/Game.test.js

```diff
@@ -190,4 +196,28 @@ describe('Move counter', () => {
     const moveCounter = getByTestId('moveCounter');
     expect(Number(moveCounter.innerHTML)).toBeGreaterThanOrEqual(1);
   });
+
+  it('should save the score to local storage', () => {
+    const { getByTestId, getByPlaceholderText } = render(<Game boardSizeX={2} boardSizeY={1} />);
+    const board = getByTestId('game-table');
+
+    // because has-sprite can be on has-user tail, we have to move two directions, to get
+    // the sprite that starts on initial user position
+    fireEvent.keyDown(board, {
+      key: 'ArrowLeft'
+    });
+    fireEvent.keyDown(board, {
+      key: 'ArrowRight'
+    });
+
+    const windowLocalStorage = global.window.localStorage;
```

Comment:
since you have mocked the localStorage, you should test if the `store` got the data. What is the need for `windowLocalStorage`